front/package.json

```diff
@@ -18,7 +18,7 @@
     "@amplitude/analytics-browser": "^2.5.2",
     "@amplitude/analytics-node": "^1.3.5",
     "@auth0/nextjs-auth0": "^3.5.0",
-    "@dust-tt/sparkle": "0.2.153",
+    "@dust-tt/sparkle": "0.2.155",
```

Comment:
You should bump the package-lock.json as well.

Reply:
I npm installed and it didn't bump it 🤷